### PR TITLE
dvm_version_dir and dvm_version_path

### PR DESCRIPTION
**Pull request by smashwilson**

### CODE REVIEW

**Comment by smashwilson** [comment]
@rgbkrk What do you think?

**Comment by rgbkrk** [comment]
Looks good, I'm not sure what the original `new` vs. `old` was for in `nvm`, so I think we can skirt that.

**Comment by smashwilson** [comment]
It looks like they changed their storage directory format at some point during the project's history, most likely when io.js forked off. We don't have to worry about that, so it simplifies a _ton_ of the code.

File: dvm.sh [hunk 1]
```diff
@@ -60,6 +72,8 @@ if [ -z "$DVM_DIR" ]; then
 fi
 unset DVM_SCRIPT_SOURCE 2> /dev/null
 
+DVM_VERSION_DIR="${DVM_DIR}/bin/docker/"
```
Comment (rgbkrk) [inline dvm.sh]: Do we even have to set this here? It seems like we only need `dvm_version_path`.

Comment (smashwilson) [inline dvm.sh]: We could probably get away without it, but it lets us not repeat the `/bin/docker/` bit, or change it later in one place.

**Pull request merged**